### PR TITLE
[MIG] crm_lead_firstname: Migration to 12.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check CIs status

**Comment by rven** [comment]
Hi @pedrobaeza,

The tests are failing because the module 'partner_firstname' of repo 'partner-contact' is not available.
I added this dependency in the module itself, but I suppose it needs to be in the repo folder?

**Comment by rven** [comment]
Tests are passing now, it was indeed a problem with the location of the oca_dependencies.txt file.

File: crm_lead_firstname/__manifest__.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Firstname and Lastname in Leads",
     "summary": "Specify split names for contacts in leads",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Customer Relationship Management",
     "website": "http://www.github.com/OCA/crm",
     "author": "Antiun Ingeniería S.L., Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline crm_lead_firstname/__manifest__.py]: Please update to the new company `Tecnativa`.

File: crm_lead_firstname/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* Rafael Blasco <rafabn@antiun.com>
```
Comment (pedrobaeza) [inline crm_lead_firstname/readme/CONTRIBUTORS.rst]: Don't put these old emails. Better to not put any mail in this case.

File: crm_lead_firstname/__manifest__.py (hunk 1 above)
Comment (rven) [reply crm_lead_firstname/__manifest__.py]: What should be updated can you insert a code suggestion? (https://thenextweb.com/dd/2018/10/16/github-launches-suggested-changes-to-make-it-easier-to-collaborate-within-pull-requests/)
I don't know what I need to change...

File: crm_lead_firstname/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (rven) [reply crm_lead_firstname/readme/CONTRIBUTORS.rst]: What should be updated can you insert a code suggestion? (https://thenextweb.com/dd/2018/10/16/github-launches-suggested-changes-to-make-it-easier-to-collaborate-within-pull-requests/)
I don't know what I need to change...

File: crm_lead_firstname/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply crm_lead_firstname/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

File: crm_lead_firstname/README.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Antiun Ingeniería S.L.
```
Comment (pedrobaeza) [inline crm_lead_firstname/README.rst]:
```suggestion
* Tecnativa
```

File: crm_lead_firstname/README.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Contributors
+~~~~~~~~~~~~
+
+* Rafael Blasco <rafabn@antiun.com>
```
Comment (pedrobaeza) [inline crm_lead_firstname/README.rst]:
```suggestion
* Rafael Blasco
```

File: crm_lead_firstname/README.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+* Jairo Llopis <yajo.sk8@gmail.com>
```
Comment (pedrobaeza) [inline crm_lead_firstname/README.rst]:
```suggestion
* Jairo Llopis
```

File: crm_lead_firstname/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (pedrobaeza) [reply crm_lead_firstname/readme/CONTRIBUTORS.rst]:
```suggestion
* Rafael Blasco
```

File: crm_lead_firstname/readme/CONTRIBUTORS.rst [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
+* Jairo Llopis <yajo.sk8@gmail.com>
```
Comment (pedrobaeza) [inline crm_lead_firstname/readme/CONTRIBUTORS.rst]:
```suggestion
* Jairo Llopis
```

**Review by pedrobaeza — APPROVED** [review]
Thanks!

**Review by tbaden — APPROVED** [review]
code review: lgtm

**Pull request merged**